drivers/generic/generic.go

```diff
@@ -16,17 +18,15 @@ import (
 
 type Driver struct {
 	*drivers.BaseDriver
-	SSHKey string
```

Comment:
Changing this field name _will_ be problematic without adding a `json:"SSHKey"` tag to indicate that in marshalling / demarshaling it should look for that field name.  Existing `config.json` for `generic` machines will break trying to look for that field.  My feeling on the matter is largely that the benefit of changing such a field name does not outweight the cost.